vtex-apps/yotpo
# Wrike 464877928: Make sure widgets are refreshed after yotpo script loads

Previously, Yotpo widgets were sometimes not initializing if user navigated to a product page. Refreshing the page would cause them to show. I have modified the `useEffect` hook calls so that they fire whenever the `data` or `product` variables change, which should eliminate the problem.

To test:
1. Go to https://yotpo--eriksbikeshop.myvtex.com
2. Navigate to a product page
3. Verify that the stars and "Write a review" link show up at the top left of the product page, and that the reviews widget shows up at the bottom of the page
4. Refresh the page and verify that the items from step 3 still appear
